Automattic/pocket-casts-android
# [Kids Profile] Add Banner to profile screen

## Description
- Add kids profile banner to profile screen
- If the user taps dismiss, the banner doesn’t show again.
- See: p1721145824361309-slack-C07CGGZR59U
- See: G60UqxPhTynbalo0Vfr1NC-fi-1853_16844

Fixes #2508

## Testing Instructions
1. Have `KIDS_PROFILE` feature flag enabled
2. Go to profile
3. ✅ Ensure you see the kids banner
4. Disable feature flag
5. ✅ Ensure you don't see the kids banner
6. Enable the feature flag again
7. Tap on dismiss button in kids banner
8. ✅ Ensure you don't see the banner anymore
9. Close the app and open again
10. ✅ Ensure you don't see the banner


## Screenshots or Screencast 
<img src="https://github.com/user-attachments/assets/8c5ff419-eebf-4557-8430-288751e00e6c" width="300">


<img width="300" src="https://github.com/user-attachments/assets/462b715c-9698-4ee2-bb24-afb5350457e0">

## Checklist
 
#### I have tested any UI changes...
